### PR TITLE
fix: set rpc to call for queryInfo in transaction/fee-estimate

### PR DESCRIPTION
**Pull request by TarikGul**
Switches the use of RPC calls for queryInfo to runtime calls for `/transaction/fee-estimate`. Will use rpc as a fallback.

### CODE REVIEW

**Review by marshacb — APPROVED** [review]
looks good

File: src/services/transaction/TransactionFeeEstimateService.ts [hunk 1]
```diff
@@ -32,9 +32,20 @@ export class TransactionFeeEstimateService extends AbstractService {
 		transaction: string
 	): Promise<RuntimeDispatchInfo> {
 		const { api } = this;
+		const apiAt = await api.at(hash);
```
Comment (IkerAlus) [inline src/services/transaction/TransactionFeeEstimateService.ts:35]: for my understanding, what is the difference of pulling the hash at this point and not inside the `try`?

**Review by IkerAlus — APPROVED** [review]
LGTM

File: src/services/transaction/TransactionFeeEstimateService.ts (hunk 1 above)
Comment (TarikGul) [reply src/services/transaction/TransactionFeeEstimateService.ts:35]: No big difference, more so for readability. The `try` is really just dealing with the async call whereas establishing the `api.at()` is declaring the runtime api.

**Pull request merged**